code/game/machinery/transformer.dm

```diff
@@ -90,15 +90,15 @@
 	sleep(5)
 
 	use_power(5000) // Use a lot of power.
-	var/mob/living/silicon/robot/R = H.Robotize()
+	var/mob/living/silicon/robot/R = H.Robotize(connect_to_ai = FALSE)
```

Comment:
This seems to make cyborgs made from the machine not connected to the ai, which uh, isn't that the point?

Reply:
If `connect_to_ai` is true `TryToConnectAI` is called. `TryToConnectAI` sets `connected_ai` to the AI with the fewest cyborgs. `set_connected_ai(masterAI)` is called later.